src/firecracker/swagger/firecracker.yaml

```
@@ -282,6 +282,30 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /hotplug:
+    put:
+      summary: Used to hot plug resources into the microVM
```

Please, use active voice here, similarly to the other endpoints.

Suggestion:

```suggestion
      summary: Hotplugs resources to the microVM.
```

Also, is this endpoint post-boot only? I assume it should be, right?

---

Yes, post-boot only. I'll add that to the summary, in line with what other endpoints have.